README.md

```diff
@@ -2,6 +2,19 @@
 Frontend + backend of web application for income + expenses planner
 
 
+### How to run the app with docker-compose:
+```
+cd backend
+mvn clean install
```

Comment:
Domyślnie chyba zakładamy, że użytkownik korzysta z linii komend. W takim przypadku `mvn` nie zadziała - zadziała jeżeli użytkownik uruchomi całość np. w IntelliJu, ale to dlatego, że IDE dopisze mu brakujące ścieżki do instalacji Mavena (o ile Maven jest zainstalowany). Natomiast `mvnw`
> allows you to run the Maven project without having Maven installed and present on the path. It downloads the correct Maven version if it's not found (as far as I know by default in your user home directory)

![mvn](https://user-images.githubusercontent.com/48785655/172372594-71917f95-b44a-4d5e-84ef-e1a2728b4cf6.png)

Reply:
![mvnw](https://user-images.githubusercontent.com/48785655/172375709-e3691d7a-cf7f-4701-9cb3-52629ed887d4.png)

Reply:
Spoko, nawet na koniec jak juz będziemy oddawać można wrzucić te jary z target na repo to nie trzeba będzie tego robic